rsfg/src/lexer.rs

```diff
@@ -2,146 +2,187 @@
 
 use crate::Token;
 
-static SPACE: &str = " \t";
-
-// Rules: A-Z,a-z
-fn is_id_1st(c: char) -> bool {
-	c >= 'A' && c <= 'z'
-}
 // A-Z or 0-9
 fn is_id(c: char) -> bool {
-	is_id_1st(c) || (c >= '0' && c <= '9')
+	match c {
+		'0'..='9' | 'A'..='z' => true,
+		_ => false
+	}
```

Comment:
nah, this matches on brackets and underscores and such. https://www.cs.cmu.edu/~pattis/15-1XX/common/handouts/ascii.html

Reply:
Ope, that's a bad mistake. Should be `'0'..='9' | 'A'..='Z' | 'a'..='z'`, sorry.